addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -196,7 +196,7 @@
                         t-att-placeholder="field.placeholder"
                         t-att-id="field.id"
                 />
-                <div class="input-group-text pe-none"><i class="fa fa-calendar"></i></div>
+                <div class="input-group-text"><i class="fa fa-calendar"></i></div>
```

Comment:
Whoa, a pe-none on the calendar icon of our form... :grimacing: To be fully stable, can't you remove it in JS instead?

Reply:
I am already doing it in the `datetimepicker_service.js`, you can see a few line above:
```js
// TODO: Remove this line and the `pe-none` class
// from templates in master
calendarIconGroupEl.classList.remove("pe-none");
```
I'll add it back, so we take 0 risk